src/modules/RelocationServer.rb

```diff
@@ -159,6 +166,9 @@ def ReadXendSettings
       true
     end
 
+    FWD_XEND_SERVICE = "xend-relocation-server".freeze
```

Comment:
It is not ready, as the xend service is not defined. At least I have not seen it.

Reply:
But it probably could be merged as in the worst case just firewalld will not be touched at all as the service until the service is defined.

Reply:
All things xend need to be removed. Xend is only supported on SLE11 and older. This and some newer bugs entered need fixing. I will probably do a new pull request with all that is about to change.